playgrounds/astro/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "astro-playground",
+  "type": "module",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "dev": "astro dev",
+    "start": "astro dev",
+    "build": "astro build",
```

Comment:
`yarn build` (root level or even in the astro workspace) gives this error:

![image](https://user-images.githubusercontent.com/45748283/208989818-0f64f4f1-7b2f-42ed-9821-b4e2b54195cd.png)

Reply:
yes, this is the same error i mentioned in the PR description. will investigate separately.